pkg/validate/security_context.go

```diff
@@ -402,8 +424,43 @@ var _ = framework.KubeDescribe("Security Context", func() {
 
 			checkNetworkManagement(rc, containerID, false)
 		})
+	})
+
+	Context("SeccompProfilePath", func() {
```

Comment:
Probably just `Seccomp`?

Reply:
I like using the litteral CRI spec name here as it doesn't test all of Seccomp here. It is literally testing the CRI spec defined options for the SeccompProfilePath:		
```&runtimeapi.LinuxPodSandboxConfig{
			SecurityContext: &runtimeapi.LinuxSandboxSecurityContext{
				SeccompProfilePath: string,
			},
		},```